include/swift/AST/Decl.h

```diff
@@ -4698,6 +4700,8 @@ class AbstractFunctionDecl : public ValueDecl, public DeclContext {
   GenericEnvironment *getLazyGenericEnvironmentSlow() const;
 
 public:
+  Identifier getName() const { return getFullName().getBaseName(); }
```

Comment:
OK, I wanted to keep the code change as small as possible and not litter the PR with renames from `getName` to `getBaseName`.